### PR TITLE
LessLethalVend

### PR DESCRIPTION
**Pull request by IData-RedactedI**
## About the PR
*Added the less lethal alternative to the freedom vend
*Machine stocks a large number of non lethal alternatives, with disruptors, tasers, and rubber bullets

## Why / Balance
This is part of the [sprite requests](https://github.com/new-frontiers-14/frontier-station-14/issues/166) 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Nonlethalvend-showcase_scaled_7x_minified

**Changelog**

:cl: [Data_Redacted]
- add: A new vending machine has arrived on frontier. Carrying much less lethal weapons, this machine is sure to become a firm favorite for those who aren't blood thirsty killers.

### CODE REVIEW

**Review by FoxxoTrystan — COMMENTED** [review]
Good work on those!

But... the Taser should not be in the list, its extremly overpowred.
Taser should be a loot kept for vaults/events/ect..., that why they were replaced with the disabler in most servers.

Tho i think the Taser is a 1 or 2 slot in emag would be good also.

**Comment by IData-RedactedI** [comment]
Good to know about the taser. Is there a anything against it holding batons?

**Comment by RealIHaveANameOfficial** [comment]
calling this an "ammovend" seems like a misnomer given the contents of the vender. Maybe rename it to "Pacivend" or something, as in "pacify"?

**Comment by IData-RedactedI** [comment]
Taser has been removed

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/lesslethalvend.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+- type: vendingMachineInventory
+  id: LessLethalVendInventory
+  startingInventory:
+    WeaponDisabler: 10
+    WeaponPistolMk58Nonlethal: 20
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/lesslethalvend.yml:5]: Add in the mosin and the double barrel bean bag filled as well as the beanbag shotgun shells

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**